website/addons/citations/provider.py

```diff
@@ -17,6 +17,17 @@ def __init__(self, provider_name):
     def serializer(self):
         pass
 
+    def check_credentials(self, node_addon):
+        valid = True
+        try:
+            node_addon.api.client
```

Comment:
Ah, I see now; accessing client for the first time will authenticate with Mendeley. Disregard the previous comment.